.github/workflows/test.yml

```diff
@@ -187,6 +187,44 @@ jobs:
           name: "${{ github.sha }}-${{ matrix.part }}-race-output"
           path: ./${{ matrix.part }}-race-output.txt
 
+  test-rosetta:
```

Comment:
Nit: Could this be its own file?

Reply:
It could, but all tests were located there so I thought it was the best position for that.

Reply:
Definitely agree with that logic. Test here is becoming generalized since it includes cosmos visor related tests as well. It's a nit since the file is becoming large for a ci file. feel free to ignore.